src/callList.lua

```diff
@@ -32,7 +30,7 @@ function CallList.create(reference, options)
 		return CallList.addIndex(t, key, returnInformation)
 	end
 
-	self.list = Util.createDummyObject(addIndex)
+	self.list = setmetatable({}, {__index = addIndex})
```

Comment:
I replaced the `createDummyObject` function call with inline code. When I saw how the original createDummyObject function was implemented, I simplified it to this. It's simple enough to just write it out in the two places it's used, don't you think?

Reply:
I would consider this more complicated then the Pythagorean theorem. Warranting a function since it is likely to be reused in other locations.

Reply:
Also you inlined it without deleting the function itself.
The function name helps describe what we are trying accomplish.

Reply:
To be honest I didn't understand what `createDummyObject` was doing until I read the code. But I will see if I can come up with a better solution than inlining it.

Reply:
We talked about this and sort of decided to leave it inlined for now. It's not awesome, but it is a common Lua idiom.